packages/icons/src/InfoIcon.tsx

```diff
@@ -0,0 +1,65 @@
+import { shadeBlack, shadeWhite } from '@littlespoon/theme/lib/colors/token'
+import styled from 'styled-components'
+
+import { getScale } from './utils/css-helpers'
+
+export interface InfoIconProps extends React.SVGAttributes<SVGElement> {
+  /**
+   * The fill of the component. Defaults to black.
+   */
+  fill?: string
+
+  /**
+   * The stroke of the component. Defaults to black.
+   */
+  stroke?: string
+
+  /**
+   * The size of the component. Defaults to "small".
+   */
+  size?: 'xsmall' | 'small' | 'medium' | 'large'
+}
+
+export default function InfoIcon({
+  fill = shadeWhite,
```

Comment:
Should we default fill to `transparent`?

Reply:
All the other icons use the shadeWhite, should I change them to transparent ?

Reply:
Ok I think the other icons should default to transparent since if there's a background color on the parent, then transparency makes it easier for the icon to fit in but we can defer for now